### PR TITLE
Updating manifest and adding WebScience

### LINKED ISSUE #86: Manifest updates
Suggestions, from updating the News and Disinformation Study manifest:
* Set `"incognito": "not_allowed"`. See: https://github.com/citp/news-disinformation-study/issues/91 
* Bump the minimum version, `"strict_min_version": "87.0" `. Same reason.
* Use the `"browser_specific_settings` manifest key, which is documented on MDN, rather than the `"applications"` key.
* Update the homepage URL and study ID from Ion to Rally.

### PR DESCRIPTION
**Pull request by jonathanmayer**
Resolves #86 and #87.

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ import "webextension-polyfill";
 
 import { Rally, runStates } from "@mozilla/rally";
 
+import * as webScience from "@mozilla/web-science";
```
Comment (rhelmer) [inline src/background.js:9]: @jonathanmayer this causes a lint failure:
`error  'webScience' is defined but never used  no-unused-vars`

Can you make it do something more interesting?

**Review by rhelmer — CHANGES_REQUESTED** [review]
Could you please rebase this to fix the integration test failure also? Thanks!

**Review by rhelmer — APPROVED** [review]
This lgtm and is an improvement over what we have now, thanks!

**Comment by jonathanmayer** [comment]
Looks like you'll have to do the merge, I don't have write privileges in this repo.

**Pull request merged**